_includes/graphql/customisation.md

```diff
@@ -0,0 +1,244 @@
+      // to sort the results of a query. Use this to
+      // limit which fields can be used to sort a query
+      // and which direction that sort can be set to.
+      sortFields?: {
+        field: string;
+        asc: boolean;
+        desc: boolean;
+      }[];
+    };
+
+    // By default, a get and find query type is created
+    // for all included classes. Use this to disable
+    // the available query types for this class.
+    query?: {
+      get?: boolean;
+      find?: boolean;
+    };
+
+    // By default, all write mutation types are 
+    // exposed for all included classes. Use this to disable
+    // the available mutation types for this class.
+    mutation?: {
+      create?: boolean;
+      update?: boolean;
+      destroy?: boolean;
+    };
+  }>
+}
+```
+
+### Set or Update Configuration
+
+We have provided a public API in ```ParseGraphQLServer``` which accepts the above JSON object for setting (and updating) your Parse GraphQL Configuration, ```setGraphQLConfig```:
+
+```typescript
+  const parseGraphQLServer = new ParseGraphQLServer(parseServer, {
+    graphQLPath: parseServerConfig.graphQLPath,
+    playgroundPath: parseServerConfig.playgroundPath
+  });
+
+  const config = {
+    // ... ParseGraphQLConfiguration
+  };
+
+  await parseGraphQLServer.setGraphQLConfig(config);       
+```
+
+### Include or Exclude Classes
+
+By default, all of your Parse classes, including the defaults such as ```Parse.User```, ```Parse.Session```, ```Parse.Role``` are added to the schema. You can restrict this using the ```enabledForClassess``` or ```disabledForClassess`` options, which accepts an array of Class names.
+
+In the following example, we limit our GraphQL schema to only expose the default ```_User``` class, along with a few custom classes:
+
+```javascript
+{
+  enabledForClasses: ["_User", "Book", "Review", "Comment"],
+  disabledForClasses: null
+}
+```
+
+In the following example, we limit our GraphQL schema by hiding some sensitive classes:
+
+```javascript
+{
+  // undefined or null results in the default behaviour, i.e. include all classes
+  enabledForClasses: undefined,
+  // override the included classes by filtering out the following:
+  disabledForClasses: [ "UserSensitiveData", "ProductOrder", "Invoice" ]
+}
+```
+
+### Input Types
+
+By default, we enrich the schema by generating a number of [Input Types](https://graphql.org/learn/schema/#input-types) for each class. This, as a healthy side-effect improves development experience by providing type-completion and docs, though the true purpose is to define exactly what fields are exposed and useable per operation type. You can provide a ```type``` setting for any or each of your classes to limit which fields are exposed:
+
+In the following example, we have a custom class called ```Review``` where the fields ```rating``` and ```body``` are allowed on the ```create``` mutation, and the field ```numberOfLikes``` on the ```update``` mutation:
+
+```json
```

Comment:
It is not a valid json and it will probably be `red` when rendered. Can you please use javascript for this one and actually all others?